shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShell.java

```diff
@@ -54,6 +57,8 @@ public static void main(String[] args) {
       System.out.println("Cannot run fsadmin shell as master hostname is not configured.");
       System.exit(1);
     }
+    // Reduce the RPC retry max duration to fall earlier for CLIs
```

Comment:
can we update the propertykey description to mention that the default is 5 seconds for CLI commands?